gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/commands/UserCommandHandler.java

```diff
@@ -57,8 +57,10 @@ public Command.Type handleType() {
 
     @Override
     public Single<UserReply> handle(UserCommand command) {
-
         UserPayload userPayload = command.getPayload();
+        if (userPayload.getUsername() == null) {
+            return Single.error(new IllegalArgumentException("Username must not be null"));
```

Comment:
**Question**: Instead of an IllegalArgumentException, I wondering if returning a UserReply bean with the ERROR status  could work. :thinking: (with log WARN to trace the reason of the error)